projects/packages/jitm/src/class-jitm.php

```diff
@@ -137,36 +137,17 @@ public function jitm_enqueue_files() {
 		if ( $this->is_gutenberg_page() ) {
 			return;
 		}
-		$min = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
-		wp_register_style(
+
+		Assets::register_script(
 			'jetpack-jitm-css',
-			plugins_url( "css/jetpack-admin-jitm{$min}.css", __FILE__ ),
-			false,
-			self::PACKAGE_VERSION .
-			'-201243242'
-		);
-		wp_style_add_data( 'jetpack-jitm-css', 'rtl', 'replace' );
-		wp_style_add_data( 'jetpack-jitm-css', 'suffix', $min );
-		wp_enqueue_style( 'jetpack-jitm-css' );
-
-		wp_enqueue_script(
-			'jetpack-jitm-new',
-			Assets::get_file_url_for_environment( 'js/jetpack-jitm.min.js', 'js/jetpack-jitm.js', __FILE__ ),
-			array( 'jquery' ),
-			self::PACKAGE_VERSION,
-			true
-		);
-		wp_localize_script(
-			'jetpack-jitm-new',
-			'jitm_config',
+			'../build/index.js',
+			__FILE__,
 			array(
-				'api_root'               => esc_url_raw( rest_url() ),
-				'activate_module_text'   => esc_html__( 'Activate', 'jetpack' ),
-				'activated_module_text'  => esc_html__( 'Activated', 'jetpack' ),
-				'activating_module_text' => esc_html__( 'Activating', 'jetpack' ),
-				'nonce'                  => wp_create_nonce( 'wp_rest' ),
```

Comment:
We still rely on those values in the JS at the moment, so we cannot just remove them I'm afraid.

Reply:
I gave this a try in 94f07b26ea8a2ec49d092ec67cfac047d0b0343a